### ISSUE TITLE
Scheduled GitHub Actions also run in forks

### ISSUE THREAD

**Issue by solardiz** [lkrg-org/lkrg#138]
This should have been obvious, but I just found out that our daily "mkosi boot (mainline)" workflows also run in forks of this repo. While this provides extra testing, we're unlikely to ever notice and benefit from such testing in the many forks, yet they consume significant resources of GitHub.

I think there are two tasks here:

1. We should somehow have this disabled in the forks - perhaps we can only do it for future/updated forks, if at all (while keeping the workflows scheduled here)? If we can't disable just for the forks, then perhaps we have to give up on the scheduled workflows here as well, as otherwise we end up abusing GitHub.
2. For workflows that run in the forks anyway (if we can't disable them from here), we may want to somehow monitor their results (e.g., in case this triggers a race condition unnoticed in our testing here).

@vt-alt What do you think we should do, and can you please help with it?

**Comment by solardiz** [comment]
Upon manual review of the forks, somehow the daily scheduled workflows run only in some. Currently, they run in:

https://github.com/solardiz/lkrg/actions
https://github.com/wladmis/lkrg/actions

Previously, they also ran in:

https://github.com/0xC0ncord/lkrg/actions
https://github.com/vt-alt/lkrg/actions

That's 4 out of 28 forks total. (GitHub's displayed fork count is 30, but I think that includes deleted forks.)

I don't know what causes them to run or not in a fork - some GitHub account setting or maybe activity in the forked repo?

So our current waste of GitHub resources isn't that bad, and we can ask the few people to re-fork if needed to stop it. However, we also/primarily need to understand/fix this issue for any future forks (which can eventually be far more numerous).

**Comment by vt-alt** [comment]
> https://github.com/vt-alt/lkrg/actions

This is not just random fork, this is where I test these features for PRs.

**Comment by vt-alt** [comment]
I just tried to fork passwdqc to see what will happen with actions there.
On the page https://github.com/vt-alt/passwdqc/actions it says to me:

"**Workflows aren’t being run on this forked repository**
Because this repository contained workflow files when it was forked, we have disabled them from running on this fork. Make sure you understand the configured workflows and their expected usage before enabling Actions on this repository.

I understand my workflows, go ahead and enable them <-- green button

View the workflows directory "

So with a plain like-fork workflows aren't enabled by default.
